mab2rec/_version.py

```diff
@@ -4,5 +4,5 @@
 
 __author__ = "FMR LLC"
 __email__ = "opensource@fmr.com"
-__version__ = "1.0.1"
+__version__ = "1.0.2"
```

Comment:
We don't need a version update, no? This isn't changing anything in the library itself.

Reply:
The PyPi package contains notebooks. So I think to update PyPi release, we would need a new version? After some search online, I think it seems ad-hoc to update pypi release with the same version though.

Reply:
When you say "PyPI package contains notebooks", how would you find that? My impression is that if you wanted to utilize these notebooks, you would clone the repo.

Reply:
Ah, sorry, my bad. PyPi doesn't have notebooks. There is no change to library source. The update is to the notebook in github repo. 

Another minor update is made to the README file, where we claim requirement of Python 3.7+. README is included in PyPI release though. Would the change be too small to get PyPI updated? @dorukkilitcioglu @skadio